components/ChatApp.tsx

```diff
@@ -279,29 +279,38 @@ export class ChatApp extends React.Component<
             }));
         });
         if (this.props.candidateName !== undefined) {
+            const channelName =
+                this.props.username + " - " + this.props.candidateName;
             // Si un candidate a été invité
             const previousChannel = this.channel || undefined;
-            try {
+            // false === channel non existant, true === channel déjà crée
+            let created = false;
+            // @ts-ignore
+            const paginator = await this.chatClient.getSubscribedChannels();
```

Comment:
Here, ts gives me an error, it thinks that getSubscribedChannels need an argument, but in twilio doc, it got no args, so i silent this warning

Reply:
try 
```js
await this.chatClient.getSubscribedChannels(undefined);
```